kde-plasma/plasma-meta/plasma-meta-6.0.49.9999.ebuild

```diff
@@ -80,12 +78,20 @@ RDEPEND="
 		>=kde-plasma/kde-gtk-config-${PV}:${SLOT}
 		x11-misc/appmenu-gtk-module
 	)
-	handbook? ( kde-apps/khelpcenter:* )
+	handbook? (
+		!webengine? ( kde-apps/khelpcenter:5 )
+		webengine? ( kde-apps/khelpcenter:6 )
+	)
```

Comment:
ah, I missed this on first pass.  perhaps we can get upstream behavior to be locally discovered manuals also